python/whylogs/extras/image_metric.py

```diff
@@ -211,6 +213,9 @@ def _update_relevant_submetrics(self, name: str, data: PreprocessedColumn) -> No
     def columnar_update(self, view: PreprocessedColumn) -> OperationResult:
         count = 0
         for image in list(chain.from_iterable(view.raw_iterator())):
+            if isinstance(image, np.ndarray):
+                image = Image.fromarray(image, mode="RGB")
```

Comment:
what about other formats with different numbers of channels? We'll need to document the assumptions here and interpretation as RGB.

We also see RGBA, and L in some of our demos for example.

Reply:
PIL's fromarray will produce an image we can profile without the mode parameter for a variety of shapes and formats, I think we can leave `mode` out, at least initially.

Reply:
Just did some manual testing and the image.mode param is correctly set based on the numpy shape for RGB, RGBA, and L, so I think we're covered here.